StreamChatClient_v3/APIClient/APIClient.swift

```diff
@@ -0,0 +1,10 @@
+//
+// APIClient.swift
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+class APIClient {
+  init(currentUser: User) {}
```

Comment:
The same as for `WebSocketClient`. The `APIClient` needs only `URLSession`. 
The `Client` is responsible to manage user and token and setup sub-services/systems.

Reply:
👍